services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/structure/directory/IndexDirectory.java

```diff
@@ -2,9 +2,9 @@
 
 package app.coronawarn.server.services.distribution.assembly.structure.directory;
 
+import app.coronawarn.server.common.shared.collection.ImmutableStack;
+import app.coronawarn.server.common.shared.functional.Formatter;
 import app.coronawarn.server.services.distribution.assembly.structure.Writable;
-import app.coronawarn.server.services.distribution.assembly.structure.util.ImmutableStack;
-import app.coronawarn.server.services.distribution.assembly.structure.util.functional.Formatter;
 import app.coronawarn.server.services.distribution.assembly.structure.util.functional.WritableFunction;
```

Comment:
Is there a reason that WritableFunction is not extracted into the shared module?

Reply:
Hi Felix. Actually, the entire "structure" package would make sense to be moved into an "io" package into shared module but I would do this in the next iteration as there are dozens of files and would really complicate this PR. What do you think ?